src/map/pc.cpp

```diff
@@ -7974,13 +7987,13 @@ int pc_resetstate(struct map_session_data* sd)
 	if (battle_config.use_statpoint_table)
 	{	// New statpoint table used here - Dexity
 		if (sd->status.base_level > MAX_LEVEL)
-		{	//statp[] goes out of bounds, can't reset!
+		{	//out of bounds, can't reset!
```

Comment:
Would it not be better to simply cap the player's level to MAX_LEVEL here?
Or rather to the maximum level depending on the job?